share/dictionary/radius/dictionary.huawei

```diff
@@ -189,12 +189,12 @@ VALUE	Port-Mirror			Enable			3
 
 ATTRIBUTE	Account-Info				184	string
 ATTRIBUTE	Service-Info				185	string
-ATTRIBUTE	DHCP-Option				187	octets
+ATTRIBUTE	DHCP-Option				187	string
 ATTRIBUTE	AVpair					188	string
 ATTRIBUTE	Delegated-IPv6-Prefix-Pool		191	string
-ATTRIBUTE	IPv6-Prefix-Lease			192	octets
-ATTRIBUTE	IPv6-Address-Lease			193	octets
-ATTRIBUTE	IPv6-Policy-Route			194	ipv6prefix # manual says string?
+ATTRIBUTE	IPv6-Prefix-Lease			192	string
```

Comment:
`string` is _printable text_.  From looking at the link, this attribute is actually a structure with multiple fields.  So this should either be left as `octets`, or it should be defined as a `struct`, and given child  `MEMBER` fields.